contracts/feature-tests/composability/interact/src/composability_interact_config.rs

```diff
@@ -0,0 +1,32 @@
+use serde::Deserialize;
+use std::io::Read;
+
+/// Config file
+const CONFIG_FILE: &str = "config.toml";
+
+/// Multisig Interact configuration
+#[derive(Debug, Deserialize)]
+pub struct Config {
+    gateway: String,
```

Comment:
according to the `config.toml`, there is no `gateway`, the deserialization will not go through

Reply:
Fixed, thanks!